metagraph/src/cli/align.cpp

```diff
@@ -290,14 +305,14 @@ std::string format_alignment(std::string_view header,
     if (!config.output_json) {
         sout += fmt::format("{}\t{}", header, paths.get_query());
         if (paths.empty()) {
-            sout += fmt::format("\t*\t*\t{}\t*\t*\t*", config.alignment_min_path_score);
+            sout += fmt::format("\t*\t*\t{}\t*\t*\t*\n", config.alignment_min_path_score);
         } else {
             for (const auto &path : paths.data()) {
                 sout += fmt::format("\t{}", path);
             }
+            sout += "\n";
```

Comment:
use `fmt::join`
```suggestion
            sout += fmt::format("\t{}\n", fmt::join(paths.data(), "\t"));
```

Reply:
I tried this, but it doesn't compile because `formatter<Alignment>` needs to be implemented. I guess if we do need to use `fmt::join` more often in the future, I'll implement it.